code/modules/mob/living/simple_animal/hostile/megafauna/hierophant.dm

```diff
@@ -68,16 +68,18 @@ Difficulty: Hard
 							   /datum/action/innate/megafauna_attack/cross_blasts,
 							   /datum/action/innate/megafauna_attack/blink_spam)
 
+	var/abilities_landed = 0 //amount of abilities that has been landed on the current target
```

Comment:
Can you make this and the ones below auto doc proper comments? Not required but a nice to have